app/src/main/res/layout/activity_launcher.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".LauncherActivity">
+
+    <TextView
+        android:id="@+id/title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/launcher_title"
+        android:layout_marginTop="@dimen/launcher_title_top_margin"
+        style="@style/Base.TextAppearance.Sensibill.H1"
```

Comment:
We should use the non-base versions here, also the file could use an autoformat.  Other than that, looks good!

Reply:
updated